src/main/resources/view/MainWindow.fxml

```diff
@@ -54,22 +55,36 @@
           <!--People left panel-->
         <StackPane minWidth="300" prefWidth="NaN" styleClass="pane-with-border" HBox.hgrow="ALWAYS" VBox.vgrow="ALWAYS">
         <VBox fx:id="personList" minWidth="340" prefWidth="340" styleClass="pane-with-border" VBox.vgrow="ALWAYS">
+          <Label styleClass="label-header" text="Students" textAlignment="CENTER" textFill="WHITE" />
+          <Label fx:id="noSelectedText" styleClass="label-subheader"
+                 text="Select a student to read." textAlignment="CENTER" />
           <padding>
             <Insets bottom="10" left="10" right="10" top="10" />
           </padding>
+          <HBox>
+            <padding>
+              <Insets bottom="5"  />
+            </padding>
+          </HBox>
           <StackPane fx:id="personListPanelPlaceholder" VBox.vgrow="ALWAYS" />
         </VBox>
         </StackPane>
           <!--Mods right panel-->
           <StackPane minWidth="300" prefWidth="NaN" styleClass="pane-with-border"
                      HBox.hgrow="ALWAYS" VBox.vgrow="ALWAYS">
-            <VBox fx:id="modList" minWidth="340" prefWidth="340" styleClass="pane-with-border" VBox.vgrow="ALWAYS">
-              <Label styleClass="label-header" text="Mods Taken" textAlignment="CENTER" textFill="WHITE" />
-              <Label fx:id="noSelectedText" styleClass="label-subheader"
-                     text="Select a student to view their module details." textAlignment="CENTER" />
-              <padding>
-                <Insets bottom="10" left="10" right="10" top="10" />
-              </padding>
+
+          <VBox fx:id="modList" minWidth="340" prefWidth="340" styleClass="pane-with-border" VBox.vgrow="ALWAYS">
+          <Label styleClass="label-header" text="Mods Taken" textAlignment="CENTER" textFill="WHITE" />
+                <Label fx:id="modDescription" styleClass="label-subheader"
+                       text="Filter students taken a mod using the find command." textAlignment="CENTER" />
```

Comment:
I think there is a grammatical error here but I think fixing it would result in the text being cut off.

Should we instead write "Find students taking a mod with the find command"?

Reply:
😅 Thanks edited!